packages/ui-popover/src/Popover/index.js

```diff
@@ -302,10 +302,15 @@ class Popover extends Component {
 
     this._handleMouseOver = handleMouseOverOut.bind(null, (event) => {
       this.show(event)
+      clearTimeout(this.mouseOutTimeout)
     })
     this._handleMouseOut = handleMouseOverOut.bind(null, (event) => {
-      this.hide(event)
+      // this is needed bc the trigger mouseOut fires before tooltip mouseOver
+      this.mouseOutTimeout = setTimeout(() => {
+        this.hide(event)
+      }, 1)
     })
+    this.mouseOutTimeout = undefined
```

Comment:
not a nice solution, im open to better ideas :)

Reply:
A nicer way would be to refactor this whole tooltip-popover-dialog-contextview family, but maybe after v8 :D